CONP_FAQ.md

```diff
@@ -1,47 +1,137 @@
+## <a name="tools"></a> Tools & Pipelines
+
+#### **What tools/pipelines are contained in CONP?**
+
+CONP contains a diverse range of XXXXXX tools and pipelines (March 2020).  Many of these tools/pipelines come from neuroscience research institutes, while others link to public resources that may be of interest to neuroscientists. A full list can be found [here](https://portal.conp.ca/pipelines).
+
+#### **What are the requirements to install and run a tool?**
+
+CONP pipelines can be easily installed and run on any computer with a container engine. You will need to install:
+
+- [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/) 
+- [Boutiques](https://pypi.org/project/boutiques/) python package (`pip install boutiques`)
+
+#### **How can I run a CONP tool/pipeline?**
+
+First, ensure your have [Boutiques](https://pypi.org/project/boutiques/) installed with [Docker](https://www.docker.com/) or [Singularity](https://singularity.lbl.gov/).
+
+If you have never used Boutiques in the past, ensure to go through the following [tutorial](https://nbviewer.jupyter.org/github/boutiques/tutorial/blob/master/notebooks/boutiques-tutorial.ipynb#reusing_tools).
+
+Tools/pipelines can be run on Boutiques based on their Zenodo ID as follows:
+
+1) The example command will create a first minimal invocation so that you don't have to start from scratch: `bosh example <zenodo_id>`. If you feel like starting with a more complete set of options, you can pass `--complete` to the example command: `bosh example --complete <zenodo_id>`
+
+2) You can now edit a JSON example invocation (`example_invocation.json`) to add your input values based on the example provided in 1)
+
+
+3) You are now all set to use the exec command to launch an analysis. Note, the option `-s` will show live stdout/stderr in the terminal.
+
+```bash
+bosh exec launch -s <zenodo_id> <invocation>
+```
+
+#### **I get the following error when running the tool on bosh**
+
+```bash
+Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
+docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.
+See 'docker run --help'.
+...
+Exit code
+125
+```
+
+=> ensure you have Docker installed and running before executing the `bosh` command.
+
+
+```bash
+</data/dataset> does not exist
+
+Traceback (most recent call last):
+  File "/run.py", line 53, in <module>
+    run('bids-validator %s'%args.bids_dir)
+  File "/run.py", line 25, in run
+    raise Exception("Non zero return code: %d"%process.returncode)
+Exception: Non zero return code: 2
+...
+Exit code
+1
+```
+
+=> ensure to mount the directory where your data is on the container by using the `-v` option in the bosh command. For example, if your data is in `/data`, then you need to specify `-v /data:/data` in the `bosh` command as follows:`bosh exec launch -s <zenodo_id> <invocation> -v <directory_to_mount>`
```

Comment:
"ensure to mount the directory where your data is"->"confirm your data directory is mounted"